app/builders/finance_report_builder.rb

```diff
@@ -12,8 +12,9 @@ class FinanceReportBuilder
                   benefit_count benefit_sum income_count income_sum]
 
   def initialize(start_date, end_date)
-    @date_from = Date.parse(start_date.to_s)
-    @date_to = Date.parse(end_date.to_s)
+    @date_from = Date.parse(start_date.to_s).to_datetime
```

Comment:
You could use the [`DateTime#parse`](http://ruby-doc.org/stdlib-2.3.0/libdoc/date/rdoc/DateTime.html#method-c-parse) method here.